test/tests/api/worker/rest/EntityRestCacheTest.ts

```diff
@@ -1709,6 +1714,240 @@ export function testEntityRestCache(name: string, getStorage: (userId: Id) => Pr
 				o(entityRestClient.setup.callCount).equals(1)
 			})
 		})
+
+		o.spec("CacheMode.Bypass", () => {
```

Comment:
not that you need to rewrite those but I generally think it's better to prepare the test data to be in the expected state and then do a short test rather then do a multi-step test that modifies the state after each state